x-pack/plugins/cases/public/components/case_action_bar/action_bar_status_item.tsx

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { type ReactNode, memo } from 'react';
+import { EuiTitle } from '@elastic/eui';
+
+interface Props {
```

Comment:
we usually include component name in Props variable name
```suggestion
interface ActionBarStatusItemProps {
```

for example like here: https://github.com/elastic/kibana/pull/177811/files#diff-d4e1bdece2e51da6b323198b4ea4fbcda89a454afc82bc4367c302cd65ac6c44R46